eventstore/tests/test_whatsapp_actions.py

```diff
@@ -201,7 +201,7 @@ def test_handle_edd_label_disabled(self):
 
         with patch("eventstore.whatsapp_actions.handle_edd_message") as handle:
             handle_inbound(message)
-            handle.assert_not_called
+            _var = handle.assert_not_called
```

Comment:
I think this was also a mistake, it should have been `handle.assert_not_called()`

Reply:
👍